nntrainer/layers/conv1d_layer.cpp

```diff
@@ -57,17 +68,10 @@ void Conv1DLayer::finalize(InitLayerContext &context) {
 
   setPropertyKV(props::FilterSize::key,
                 std::to_string(std::get<props::FilterSize>(conv_props).get()));
-  setPropertyKV(
-    props::KernelSize::key,
-    "1," + std::to_string(std::get<props::KernelSize>(conv_props).get()));
-  setPropertyKV(props::Stride::key,
-                "1," +
-                  std::to_string(std::get<props::Stride>(conv_props).get()));
-  setPropertyKV(props::Padding2D::key,
-                std::get<props::Padding2D>(conv_props).get());
-  setPropertyKV(props::Dilation::key,
-                "1," +
-                  std::to_string(std::get<props::Dilation>(conv_props).get()));
+  setPropertyKV(props::KernelSize::key, "1," + std::to_string(kernel_size));
+  setPropertyKV(props::Stride::key, "1," + std::to_string(stride));
+  setPropertyKV(props::Padding2D::key, padding_str);
```

Comment:
I wonder if it is right. It does not consistent with line 103. Here, property is Padding2D, but line 103, it is Padding1D. Please check once again.

Reply:
For such issues described in https://github.com/nnstreamer/nntrainer/pull/1951#discussion_r917596381, conv1d needs Padding1D, not the Padding2D. (That is the reason for Padding1D at line 103)

But for now conv1d uses conv2d layer so we need to convert Padding1D property to Padding2D property. (line 73)

Reply:
right. thanks.